File: src/types/employee.ts
export type Employee = {
    id: number;
    name: string;
    phone: string;
    email: string;
    registration_number: string;
    position: string;
    work_unit: string;
    status: 1 | 2; // 1. Aktif, 2. Tidak Aktif
}

File: backend/models/employee.go
package models

type Employee struct {
	ID                 uint   `json:"id"`
	Name               string `json:"name"`
	Phone              string `json:"phone"`
	Email              string `json:"email"`
	RegistrationNumber string `json:"registration_number"`
	Position           string `json:"position"`
	WorkUnit           string `json:"work_unit"`
	Status             int    `json:"status"`
}
